src/utils/helpers.coffee

```diff
@@ -64,6 +65,15 @@ exports.excludeFile = (fileName, options) ->
             # Only instrument files that are inside the project.
             excluded = true
 
+        # For each exclude value try to use it as a pattern to exclude files
+        exclude.map (pattern) ->
+            glob.sync pattern,
```

Comment:
Although I'm 90% through making this work by just doing one call to glob.  Haha.  :P  I also made one very small change - historically this would let you specify an include starting with a "/" where / was the project root.  This works the same as, say, .gitignore or .npmignore.  If we pass in `root: basePath` in addition it `cwd: basePath`, then we get exactly this behavior - the leading / is effectively optional.  I'm also going to strip out the old style "prefix match" code - if we're going to make a breaking change, may as well break it good.  :)